.travis.yml

```diff
@@ -99,10 +112,12 @@ before_install:
   
 script:
     - cd build
-    - cmake -DCMAKE_BUILD_TYPE=Debug -DENABLE_COVERAGE=${COVERAGE} -DPISA_COMPILE_HEADERS=${PISA_COMPILE_HEADERS} ${CMAKE_OPTIONS} ..
+    - cmake -DCMAKE_BUILD_TYPE=Debug -DENABLE_COVERAGE=${COVERAGE} -DPISA_COMPILE_HEADERS=${PISA_COMPILE_HEADERS} -DPISA_ENABLE_CLANG_TIDY=${TIDY} -DPISA_CLANG_TIDY_EXECUTABLE=clang-tidy-9 ${CMAKE_OPTIONS} ..
     - if [[ "$PISA_COMPILE_HEADERS" != "On" && "$CLANG_FORMAT" != "On" ]]; then
         make -j2;
-        CTEST_OUTPUT_ON_FAILURE=TRUE ctest -j2;
+        if [[ "$TIDY" != "On" ]]; then
```

Comment:
What’s this for?

Reply:
"Don't run tests if running `clang-tidy`" I figured it might make sense not to push our testing capabilities. I can turn it back on, but that means additional 15 min or so.

Reply:
But aren’t you running tests? Isn’t that ctest command inside the branch?